docs-2.0/7.data-security/2.backup-restore/4.br-restore-data.md

```diff
@@ -2,19 +2,23 @@
 
 If you use the BR to back up data, you can use it to restore the data to Nebula Graph. This article introduces how to use the BR to restore data from backup files.
 
+> **NOTE**: The restoration process is performed OFFLINE.
+
+> **CAUTION**: During the restoration process, the data on the target Nebula Graph cluster is removed and then is replaced with the data from the backup files. If necessary, back up the data on the target cluster.
+
 ## Prerequisites
 
 To restore data with the BR, do a check of these:
 
 - The BR is compiled. For more information, see [Compile BR](2.compile-br.md).
 
-- The Nebula Graph services are running and we recommend that the restoration is performed when the application request traffic is very low.
+- No application is connected to the targte Neubla Graph cluster.
```

Comment:
target

Reply:
Besides. AFAIK, the target nebula graph cluster should be shutdown

Reply:
> target

and Nebula. So many thanks.